packages/tools/package/test/module/package/package.test.js

```diff
@@ -101,15 +100,17 @@ describe('Package', () => {
 
       it('should attempt to ammend the package definition file to the package path', () => pack.apply()
         .then(() => {
-          expect(spies.path.join).toHaveBeenCalledOnceWith(
+          expect(spies.path.join).toHaveBeenCalledTimes(1);
```

Comment:
We have extended jest in packages/webex/calling to add the toBeCalledOnceWith() to combine both the commands. Do you think we could add this to the config you've created for jest in this PR and consume it here and other places?

Reply:
I'm a bit hesitant to utilize a custom BDD clause, as it we'd move away from a standard configuration to a more complex one.

**BUT**, we could definitely add this to the `./config/jest/additions/**` folder.

We should discuss this route and how to make it **expandable** and **readable** in a meeting.

Reply:
Sure @InteractiveTimmy , let's discuss this in a meeting next week